src/styles/_config.scss

```diff
@@ -0,0 +1,5 @@
+@forward 'config/fonts';
+@forward 'config/functions';
+@forward 'config/mixins';
+@forward 'config/resets';
```

Comment:
`resets` must be imported first, otherwise you end up overriding everything above it.